### ISSUE TITLE
🚀 Feature: 

### ISSUE THREAD

**Issue by LenaDooms** [novuhq/novu#3270]
### 🔖 Feature description

Uncancel cancelled events

### 🎤 Why is this feature needed ?

There are scenarios where an email was cancelled but should be sent after all. Instead of needing to make a new email, it would be nice to be able to change the status of a cancelled event

### ✌️ How do you aim to achieve this?

Based on `transactionId`, change the status of a triggered event

### 🔄️ Additional Information

To use a mail without storing the transactionId, we calculate a hash of the payload and use that as transactionId. If the payload changes, the original mail is cancelled. However, if the payload is changed back to it's original value, there's no way to send an email with that transactionId again. (Tried POST but this seems to do nothing)

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

None

**Comment by gitstart** [comment]
@LenaDooms I would like to pick this up cc @scopsy

**Comment by LenaDooms** [comment]
Oh great, thank you!

**Comment by davidsoderberg** [comment]
Yeah as long as the scheduled time have not passed it should only be doing the opposite of this:
https://github.com/novuhq/novu/blob/next/apps/api/src/app/events/usecases/cancel-delayed/cancel-delayed.usecase.ts#L6
